### PR TITLE
Change the named property visibility algorithm so that existing own props always make corresponding named props invisible.

### PR DESCRIPTION
**Pull request by bzbarsky**
Fixes <https://github.com/heycam/webidl/issues/152>.

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -12201,16 +12201,14 @@ depending on whether the [{{OverrideBuiltins}}]
...
-        on |O|, then return false.
-    1.  If |O| implements an [=interface=] with
-        an [{{Unforgeable}}]-annotated [=attribute=]
-        whose [=identifier=] is |P|, then return false.
     1.  If |P| is not a [=supported property name=]
         of |O|, then return false.
+    1.  If |O| has an own property named |P|, then return false.
+          <div class="note">
+            Note that this will include cases in which an object has unforgeable properties, because in practice those are always set up before objects have any supported property names, and once set up will make the corresponding named properties invisible.
+          </div>
```
Comment (domenic) [inline index.bs]: These three lines can be replaced with `Note: This will include....`

Comment (bzbarsky) [inline index.bs]: Done, thanks.

**Comment by domenic** [comment]
(In a few meetings, will look at the substantive stuff in an hour or two...)

**Comment by bzbarsky** [comment]
@yuki3, @cdumez, @travisleithead, any remaining concerns here?

**Comment by domenic** [comment]
This LGTM. It might be worth adding a note in the commit message that this fixes #152 by choosing option (1).

**Comment by bzbarsky** [comment]
> It might be worth adding a note in the commit message that this fixes #152 by choosing option (1).

Will do.

**Comment by yuki3** [comment]
I have no concern here.  LGTM.

**Pull request merged**